Fixed some build issues, improved interface of build progress (#713)
V4Fire/Client: CHANGELOG.md

_Note: Gaps between patch versions are faulty, broken or test releases._

## v3.21.0 (2022-05-13)

#### :boom: Breaking Change

> * Removed `simpleProgressWebpackPlugin` with custom progress plugin

Reviewer: ```suggestion
* Replaced `simpleProgressWebpackPlugin` to a custom progress plugin
```

#### :house: Internal

> * Removed comments from css files into dev build

Reviewer: Формально это br

```suggestion
* Now all comments are removed from compiled CSS files by default
```

> * Now all comments are removed from compiled CSS files by default

Reviewer: Это лучше перенести в Breaking Change

> * Removed deadlock for build while snakeskin waits artifacts

Reviewer: Перенести в секцию #### :bug: Bug Fix

```suggestion
* Removed deadlock during execution of Snakeskin templates `i-static-page/modules/ss-helpers`
```

> * Added names for build processes

Reviewer: ```suggestion
* Added human readable names for all registered webpack process `build/webpack`
```

> * Added human readable names for all registered webpack process `build/webpack`

Reviewer: Это лучше в Internal

> * Removed generation of redunant artifacts (`license.txt`, empty entry files) `build`

Reviewer: Это BR
